source/reference/minio-mc-admin/mc-admin-rebalance.rst

```diff
@@ -0,0 +1,151 @@
+.. _minio-mc-admin-rebalance:
+
+======================
+``mc admin rebalance``
+======================
+
+.. default-domain:: minio
+
+.. contents:: Table of Contents
+   :local:
+   :depth: 2
+
+.. mc:: mc admin rebalance
+
+Permission
+----------
+
+This command requires that the user performing it have the :policy-action:`admin:Rebalance` :ref:`policy action <minio-policy>` for the deployment.
+
+Description
+-----------
+
+.. start-mc-admin-rebalance-desc
+
+The :mc-cmd:`mc admin rebalance` command allows for objects to move from pools with less free space to pools with more free space, rebalancing the storage across pools.
```

Comment:
```suggestion
The :mc-cmd:`mc admin rebalance` command start, monitor, or stop a rebalancing operation on a MinIO deployment. Rebalancing redistributes objects across all pools in the deployment.
```

Reply:
"allows" sounds more passive, this is an active operation (is it a blocking one?)

Reply:
Reworked, checking on wither a rebalance blocks other activity.

Reply:
Rebalance runs in parallel, but recommendation is to run only with available cycles. Noted in doc.